### DISCUSSION TITLE
Suggestions for improving the NWT Reading App

### DISCUSSION THREAD

**Discussion by AngelAcuna2000** [searchwork/nwt-reading#67] [Support] [answered]
Dear developers,

I’m a user of your NWT Reading app, and I really enjoy it. It’s a great tool for studying the Bible and keeping track of my progress. I have some ideas for improving the app that I would like to share with you.

It would be great if the app could allow users to check off the chapters that they have read each day and also record the verse number that they left off at in each chapter. This way, users can easily resume their reading from where they stopped and keep track of their progress.

It would also be nice if the app could show users how many days they are behind, ahead, or if they are on schedule based on the reading speed that they selected. This would motivate users to catch up or maintain their reading pace and help them plan their reading time better.

Lastly, I think it would be helpful if the app could have a setting option that allows users to choose whether or not they want to check off all the previous chapters as read when they check off a later chapter. This way, users can either use this shortcut or avoid being asked every time if they want to mark all the previous chapters as read.

I hope you will consider these suggestions and let me know what you think. Thank you for your hard work and dedication to this project.

Sincerely, Angel Acuna

**Comment by wfleischer** [comment, marked as answer]
Dear Angel,

Thank you so much for your thoughtful suggestions and your patience. I am happy to hear that you find the NWT Reading app useful for your Bible study and progress tracking.

I wanted to let you know that the app has been completely rewritten and is now available for Android. The iOS version will be released in the upcoming weeks.

Regarding your suggestion to check off chapters and record the verse number, I feel that this is best done using a bookmark in the JW Library app.

As for showing how many days users are behind, ahead, or on schedule based on their selected reading speed, this feature is available through the badge on the schedule title and the separator lines in the schedule.

Lastly, for the option to check off all previous chapters as read when checking off a later chapter, you can check off any future section or uncheck any previous section, and you will be asked if you would like to apply that setting to all previous sections.

We appreciate your feedback and are always looking for ways to improve the app. Thank you for your support and dedication to this project.

Sincerely,
Werner Fleischer
